src/utils/status/vm/constants.js

```diff
@@ -4,6 +4,7 @@ export const VM_STATUS_STARTING = 'VM_STATUS_STARTING';
 export const VM_STATUS_VMI_WAITING = 'VM_STATUS_VMI_WAITING';
 export const VM_STATUS_IMPORTING = 'VM_STATUS_IMPORTING';
 
+export const VM_STATUS_CONVERSION_IN_PROGRESS = 'VM_STATUS_CONVERSION_IN_PROGRESS';
```

Comment:
what is this status for? Who sets it?

Reply:
> what is this status for? Who sets it?

:cry: My bad, removing,

 :hibiscus: the string is `"VM_STATUS_CONVERSION_IN_PROGRESS"` the constant is `VM_STATUS_V2V_CONVERSION_IN_PROGRESS`